docs/developer/developer-handbook.md

```diff
@@ -3,13 +3,22 @@
 <!-- TOC -->
 * [Developer's Handbook](#developers-handbook)
   * [Introduction](#introduction)
+    * [Terminology](#terminology)
   * [Building a distribution](#building-a-distribution)
-    * [Seed some data](#seed-some-data)
-    * [Perform a data transfer](#perform-a-data-transfer)
+    * [Perform a simple data transfer](#perform-a-simple-data-transfer)
+    * [Transfer some more data](#transfer-some-more-data)
   * [Core concepts](#core-concepts)
   * [The control plane](#the-control-plane)
     * [API objects in detail](#api-objects-in-detail)
+      * [Assets](#assets)
+      * [Policies](#policies)
+      * [ContractDefinitions](#contractdefinitions)
```

Comment:
I would separate terms with a whitespace, because it's not referring to the class name but the domain term (so `Contract Definitions`, `Contract Negotiations`, ...)

Reply:
they'd have to be lower case then, i.e. `Contract definitions`, but yeah, sounds good.